recipe/meta.yaml

```diff
@@ -1,176 +1,131 @@
-{% set version = "0.3.21" %}
-{% set build_number = "0" %}
-{% set sha_hash = "f36ba3d7a60e7c8bcc54cd9aaa9b1223dd42eaf02c811791c37e8ca707c241ca" %}
+{% set version = "0.3.29" %}
+{% set build_number = 0 %}
 
+# ensure INTERFACE64 variable gets detected by conda-build
+# [INTERFACE64 == "foobar"]
```

Comment:
Is it needed here or can it be removed?

Reply:
Unfortunately it looks like it is needed:

```
    ValueError: Variant configuration errors in /var/folders/my/fnvhhdw53lq0ztwwqdlx9zw00000gp/T/tmp5dalvwvc/info/recipe/conda_build_config.yaml:
      zip_key entry name_suffix in group frozenset({'name_suffix', 'SYMBOLSUFFIX', 'INTERFACE64'}) does not have any settings
      zip_key entry SYMBOLSUFFIX in group frozenset({'name_suffix', 'SYMBOLSUFFIX', 'INTERFACE64'}) does not have any settings

```

Reply:
This seems to only happen locally... But I lean towards leaving it to save someone the hassle of troubleshooting this later.

Reply:
Edit: This error seems un-related to that change and something with my local environment. I'll remove the comments as I don't think they make sense.